package-metadata.json

```diff
@@ -0,0 +1 @@
+{"dependencies": [], "version": "2015.10.18.15.47.41", "platforms": ["*"], "description": "AVR ASM syntax definition for Sublime Text 2", "url": "https://github.com/vyivanov/AVR-ASM-Sublime", "sublime_text": "*"}
```

Comment:
package-metadata.json file is created by package control manager during package installation. You should not put it under version control.